<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCDeviceList</class>
 <widget class="QWidget" name="VSCDeviceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/devicelist.png</normaloff>:/action/resources/devicelist.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/device/resources/computer.png</normaloff>:/device/resources/computer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCamera">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to add Camera</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/device/resources/camera.png</normaloff>:/device/resources/camera.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/device/resources/recorder.png</normaloff>:/device/resources/recorder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSurveillance">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to add VSCView</string>
         </property>
         <property name="statusTip">
          <string>2</string>
         </property>
         <property name="whatsThis">
          <string>3</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/surveillance.png</normaloff>:/action/resources/surveillance.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/search.png</normaloff>:/action/resources/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/videosearch.png</normaloff>:/action/resources/videosearch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VSCDeviceTrash" name="pushButtonTrash">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/user-trash-2.png</normaloff>:/action/resources/user-trash-2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="VSCDeviceTree" name="treeWidget">
       <property name="indentation">
        <number>30</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>VS Cloud</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Localhost</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/computer.png</normaloff>:/device/resources/computer.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Camera</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/camera.png</normaloff>:/device/resources/camera.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recorder</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/recorder.png</normaloff>:/device/resources/recorder.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>VSCDeviceTree</class>
   <extends>QTreeWidget</extends>
   <header>vscdevicetree.h</header>
  </customwidget>
  <customwidget>
   <class>VSCDeviceTrash</class>
   <extends>QPushButton</extends>
   <header>vscdevicetrash.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
